ansible/roles/runner/files/prci.service

```diff
@@ -10,6 +10,9 @@ WorkingDirectory=/root/freeipa-pr-ci
 ExecStart=/bin/bash -c '/root/freeipa-pr-ci/github/prci.py "$(hostname -s)" --config /root/.config/freeipa-pr-ci/config.yml'
 StandardOutput=syslog
 StandardError=syslog
+Restart=on-failure
+RestartSec=10m
```

Comment:
There is some reason behind the magical `10m` value?

Reply:
Not really, 10 minutes seemed like an appropriate time to reattempt restart. In case the service is unreachable, there's no need to try more often than that. In cases there's some other error which depletes some API requests, it seems like limiting it to 6 times per hour is reasonable so it doesn't affect other runners.

Reply:
Nice, I was just wondering if I was missing the why.